tests/Feature/Users/Registration/AccountCreationTest.php

```diff
@@ -146,7 +146,8 @@ public function testValidEmail(): void
             'email' => 'test@invalid.com',
         ]);
 
-        $this->assertNull(json_decode($response->getContent(), true));
+        // Assert empty response.
+        $this->assertEquals([], json_decode($response->getContent(), true));
```

Comment:
There's an `assertEmpty` in phpunit which would work here as well.

Reply:
Addressed in [8b104c0](https://github.com/iFixit/restarters/pull/7/commits/8b104c0c998ca5cec39e9c82e4bcc9cc6f78d4c8)